sat/cli/bootprep/image.py

```diff
@@ -208,7 +208,15 @@ def handle_existing_images(ims_client, input_images, overwrite, skip, dry_run):
         return input_images
     elif skip:
         LOGGER.info(msg_template, {'action': 'skipped'})
-        LOGGER.info('Any images which depend on these images will also be skipped.')
+
+        # Remove already existing images as dependencies of other images. They can be built right away
+        for existing_image in existing_input_images:
+            existing_image.skip = True
+            for image in input_images:
+                if existing_image in image.dependencies:
+                    LOGGER.info(f'Removing skipped, already existing {existing_image} '
```

Comment:
I feel like this wording is a little bit awkward but I can't think of a better suggestion, wouldn't hold up the merge for it.

Reply:
I think this message is not super important to the user, so I will drop it to debug level.